var/spack/repos/builtin/packages/qscintilla/package.py

```diff
@@ -130,3 +133,8 @@ def make_qsci_python(self):
             makefile = FileFilter(join_path("build", "Makefile"))
             makefile.filter("$(INSTALL_ROOT)", "", string=True)
             make("install", "-C", "build/")
+
+    @run_after("install", when="+python")
+    def build_test(self):
```

Comment:
Should this only be done if testing is requested? We could also move it to a post-install test.

Reply:
I don't think testing should be requested. This is necessary, if a python package is built, one should be able to import it. This could have saved me a lot of headaches earlier.

Reply:
> We could also move it to a post-install test.

Not sure what a 'post-install test' is exactly.

Reply:
> I don't think testing should be requested.

This is how all other Spack packages work. I don't think qscintilla should be different.

> We could also move it to a post-install test.

Not sure what a 'post-install test' is exactly.

See https://spack.readthedocs.io/en/latest/packaging_guide.html#checking-an-installation. I'm specifically referring to "stand-alone tests" as opposed to "build-time tests". All Python libraries in Spack add a "stand-alone test" that attempts to import all Python files. Qscintilla should follow suit.